src/App.tsx

```diff
@@ -6,15 +6,15 @@ import './scss/styles.scss'
 import LocalStorage from './local-storage'
 
 const App = () => {
-  useEffect(() => {
-    axios.interceptors.request.use((config: any) => {
-      const token = LocalStorage.getToken()
-      if (token) {
-        config.headers['Authorization'] = `Bearer ${token}`
-      }
-      return config
-    })
-  }, [])
+  // useEffect(() => {
+  //   axios.interceptors.request.use((config: any) => {
```

Comment:
you can take advantage of `config` object for the token removal. Here, `config` contains information about the request being sent, in which I believe there is a property `url` inside, and you can use this `url` to do conditioning